redis.conf

```diff
@@ -605,8 +605,8 @@ repl-diskless-sync yes
 # it entirely just set it to 0 seconds and the transfer will start ASAP.
 repl-diskless-sync-delay 5
 
-# When diskless replication is enabled with a delay.
-# The replication can start before the maximum delay is reached if the maximum
+# When diskless replication is enabled with a delay, it is possible to let
+# the replication to start before the maximum delay is reached if the maximum
```

Comment:
```suggestion
# When diskless replication is enabled with a delay, it is possible to let
# the replication start before the maximum delay is reached if the maximum
```

I suggested "possible to configure ... to" but with "possible to let" there is no "to".

Reply:
yes. i took your suggestion and further modified it.. but messed up. thanks,